File: tourneyAPI/Models/Entities/Characters/Daisy.cs
namespace Entities;

using Enums;

// Defines the competitive profile metadata for this playable character.
public class Daisy : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public Daisy()
    {
        Id = CharacterId.Daisy;
        characterName = CharacterName.DAISY;
        archetype = Archetype.MIX_UP;
        fallSpeed = FallSpeed.FLOATY;
        weightClass = WeightClass.LIGHTWEIGHT;
        tierPlacement = TierPlacement.S;
    }

}

File: tourneyClient/src/models/entities/Characters/Daisy.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const Daisy: Character =
{
    id: CharacterId.Daisy,
    characterName: CharacterName.DAISY,
    archetype: Archetype.MIX_UP,
    fallSpeed: FallSpeed.FLOATY,
    weightClass: WeightClass.LIGHTWEIGHT,
    tierPlacement: TierPlacement.S
}

export default Daisy; 